internal/rpc/core/events.go

```diff
@@ -165,8 +165,11 @@ func (env *Environment) Events(ctx context.Context,
 		maxItems = 100
 	}
 
+	const minWaitTime = 1 * time.Second
```

Comment:
Is there a reason these minimum/maximum wait times are not documented for this function?

Reply:
They're an implementation detail, and clients should not be written assuming specific values for them. The service is allowed to change them at will.